superset/assets/spec/helpers/setupSupersetClient.js

```diff
@@ -0,0 +1,12 @@
+import fetchMock from 'fetch-mock';
+import { SupersetClient } from '@superset-ui/core';
+
+export default function setupSupersetClient() {
+  // The following are needed to mock out SupersetClient requests
+  // including CSRF authentication and initialization
+  // global.FormData = window.FormData;
```

Comment:
going to remove this file and add it as a cherry-pick to share across these ajax refactors. this line is also needed.